indico/modules/rb_new/client/js/common/timeline/TimelineContent.module.scss

```diff
@@ -156,10 +156,6 @@ $split-row-height: $row-height/2;
 
 
 .daily-timeline {
-    margin-right: 2em;
-    display: flex;
-    flex-direction: column;
-    flex: 1;
```

Comment:
Removing `flex` and `flex-direction` is breaking the edit booking timeline.
![image](https://user-images.githubusercontent.com/8863238/54985874-f46b7700-4fb1-11e9-86b6-65d63bb5d11a.png)

Reply:
Good catch.